recipes/odbc/all/patches/missing-declarations.patch

```diff
@@ -0,0 +1,53 @@
+--- a/DriverManager/SQLError.c
++++ b/DriverManager/SQLError.c
+@@ -184,7 +184,7 @@
+ 
+ static char const rcsid[]= "$RCSfile: SQLError.c,v $ $Revision: 1.11 $";
+ 
+-static SQLRETURN extract_sql_error( EHEAD *head,
++static SQLRETURN cci_extract_sql_error( EHEAD *head,
```

Comment:
Can you explain this patch?
You're adding declarations to the header file, and add a `cci_` prefix to the implementation.

Reply:
It's not the same function (arguments are different). This function is an internal function (static) of this compilation unit.
The implementation of the function declared in ̀drivermanager.h` and visible to others compilation units is in another file.

Reply:
If it's internal and the renaming doesn't cause any errors, are these used?
You didn't change it calling sites.

Reply:
Not renaming these functions after adding missing declarations in drivermanager.h leads compilation errors (multiple declarations because these files also include drivermanager.h).

You can check the last version of unixODBC, it's fixed more or less like this.

Reply:
Thanks for the explanation.